patches/api/0009-Add-Position.patch

```diff
@@ -415,6 +423,11 @@ index 7c4db051472fb6a6c6d24092dc6f75487356690a..3b99f359f556e6f2c341d55fa69b7462
 +    }
 +
 +    @Override
++    public boolean isFinite() {
++        return io.papermc.paper.math.FinePosition.super.isFinite() && NumberConversions.isFinite(this.getYaw()) && NumberConversions.isFinite(this.getPitch());
```

Comment:
You can use the java equivalent with Float.isFinite

Reply:
Fixed.